packages/components/src/DateTimePickers/InputDateRangePicker/InputDateRangePicker.component.js

```diff
@@ -0,0 +1,145 @@
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
+import uuid from 'uuid';
+import classnames from 'classnames';
+import { Popper } from 'react-popper';
+
+import FocusManager from '../../FocusManager';
+import { focusOnCalendar } from '../../Gesture/withCalendarGesture';
+import Icon from '../../Icon';
+
+import DateRange from '../DateRange';
+import { DateRangeContext } from '../DateRange/Context';
+import useInputPickerHandlers from '../hooks/useInputPickerHandlers';
+
+import getDefaultT from '../../translate';
+
+import theme from './InputDateRangePicker.scss';
+
+const PROPS_TO_OMIT_FOR_INPUT = ['id', 'dateFormat', 'required', 'value', 'onBlur', 'onChange'];
+
+export default function InputDateRangePicker(props) {
+	const popoverId = `date-range-picker-${props.id || uuid.v4()}`;
+
+	const startDateInputRef = useRef(null);
+	const endDateInputRef = useRef(null);
+	const containerRef = useRef(null);
+	const [inputRef, setInputRef] = useState(null);
```

Comment:
keep a record of which input is focused on, and pass it to `DateRange.Picker` so it will know which picker to render